bcda/web/api_v2.go

```diff
@@ -0,0 +1,239 @@
+package web
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/CMSgov/bcda-app/bcda/models/postgres"
+
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	fhirmodels "github.com/eug48/fhir/models"
+
+	"github.com/go-chi/chi"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/CMSgov/bcda-app/bcda/database"
+	"github.com/CMSgov/bcda-app/bcda/models"
+	"github.com/CMSgov/bcda-app/bcda/responseutils"
+	"github.com/CMSgov/bcda-app/bcda/servicemux"
+	"github.com/CMSgov/bcda-app/bcda/utils"
+)
+
+// var (
```

Comment:
What do you think about moving these all to a separate directory/package? So we would have `bcda/web/v2/api.go`?

That way we keep all of the resources/constants decoupled and we support _semantic import versioning_ which is referenced in the official [Go docs](https://blog.golang.org/versioning-proposal).

We'd want to make the handlers exported but then we can drop the v2 naming suffix.